test/database.ts

```diff
@@ -295,43 +327,30 @@ describe('Database', () => {
       assert(database.formattedName_, formattedName);
     });
 
-    it('should create a SessionPool object', () => {
-      assert(database.pool_ instanceof FakeSessionPool);
-      assert.strictEqual(database.pool_.calledWith_[0], database);
-      assert.strictEqual(database.pool_.calledWith_[1], POOL_OPTIONS);
-    });
-
-    it('should accept a custom Pool class', () => {
```

Comment:
Again there is no change in the constructor of database. So why this is deleted ?

Reply:
Yeah, this test can be for the both the classes Database and Session Factory. Thanks for noticing it

Reply:
updated